src/org/opendatakit/briefcase/ui/reused/transfer/TransferFormsTableView.java

```diff
@@ -72,6 +74,7 @@
     setFillsViewportHeight(true);
 
     TableRowSorter<TransferFormsTableViewModel> sorter = sortBy(getModel(), FORM_NAME_COL, ASCENDING);
+    sorter.setComparator(SELECTED_CHECKBOX_COL, (Comparator<Boolean>) UI::compareSelectionButton);
```

Comment:
I believe we need to set another comparator for the details column as in:
```
sorter.setComparator(DETAIL_BUTTON_COL, (Comparator<DetailsStatusButton>) DetailsStatusButton::compare);

Reply:
See earlier comment on Comparable